translate/canon/Test.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright(c) 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+//
+
+using System;
+using System.Collections.Generic;
+using Google.Cloud.Translation.V2;
+using Xunit;
+
+public class TranslateTest
+{
+    readonly TranslateSample _sample = new TranslateSample();
+
+    [Fact]
+    public void TestTranslateText()
+    {
+        string translatedText = _sample.TranslateText();

```

Comment:
I like these tests a lot more than command line tests. What do you propose to do in cases when the API returns complex object?

Reply:
Return the whole complex object.  I think one of the samples I wrote does that.